openedx/core/lib/log_utils.py

```diff
@@ -0,0 +1,36 @@
+"""
+Helper functions for logging.
+"""
+import logging
+
+
+log = logging.getLogger(__name__)
+
+
+def audit_log(name, **kwargs):
+    """DRY helper used to emit an INFO-level log message.
```

Comment:
Nit: there was some back-and-forth on this, but where we landed was that the first line of the docstring should not be on the same line as the ``"""``:

http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings